addons/stock/models/stock_storage_category.py

```diff
@@ -70,3 +70,33 @@ def _domain_product_id(self):
         ('unique_product', 'UNIQUE(product_id, storage_category_id)', 'Multiple capacity rules for one product.'),
         ('unique_package_type', 'UNIQUE(package_type_id, storage_category_id)', 'Multiple capacity rules for one package type.'),
     ]
+
+    def _get_display_name_attributes(self):
+        """
+            Adds the storage capacity attributes to compose the display name
+        """
+        self.ensure_one()
+        attributes = []
+        if self.product_id:
+            attributes.append(_("Product: ") + self.product_id.name)
+        if self.package_type_id:
+            attributes.append(_("Package: ") + self.package_type_id.name)
+        return attributes
+
+    @api.model
+    def _compute_display_name_depends(self):
+        return [
+            "storage_category_id.name",
+            "quantity",
+            "product_id.name",
+            "package_type_id.name",
+        ]
+
+    @api.depends(lambda self: self._compute_display_name_depends())
```

Comment:
no need to do that for inheritance you can just override the function add your dependencies and return the super result

Reply:
Indeed, but is this new from v16 as before, you should use lambda function to extend the depends arguments ?

Reply:
But, in the extending module, you need to call one more time the compute function